src/NServiceBus.Core/DelayedDelivery/DoNotDeliverBefore.cs

```diff
@@ -8,22 +8,22 @@
     public class DoNotDeliverBefore : DelayedDeliveryConstraint
     {
         /// <summary>
-        /// The actual time when the message can be available to the recipient.
-        /// </summary>
-        public DateTime At { get; }
-
-        /// <summary>
-        /// Initializes a new insatnce of <see cref="DoNotDeliverBefore"/>.
+        /// Initializes a new insatnce of <see cref="DoNotDeliverBefore" />.
         /// </summary>
         /// <param name="at">The earliest time this message should be made available to its consumers.</param>
         public DoNotDeliverBefore(DateTime at)
         {
-            if (at <= DateTime.UtcNow)
+            if (at.ToUniversalTime() <= DateTime.UtcNow)
```

Comment:
Just for reference

https://msdn.microsoft.com/en-us/library/system.datetime.touniversaltime(v=vs.110).aspx